pkg/util/ovs/ovs.go

```diff
@@ -162,7 +163,7 @@ func (ovsif *ovsExec) exec(cmd string, args ...string) (string, error) {
 }
 
 func (ovsif *ovsExec) AddBridge(properties ...string) error {
-	args := []string{"add-br", ovsif.bridge}
+	args := []string{TIMEOUT, "add-br", ovsif.bridge}
```

Comment:
you should just add the timeout in `exec()` (like we already do with `-O OpenFlow13` for `ovs-ofctl`)

Reply:
+1 on moving timeout to exec()

Reply:
@danwinship PTAL, thanks!

Reply:
@dcbw Don't we need this timeout for ovs-ofctl (I don't know which db it hits)?